### ISSUE TITLE
Active learning for semantic Segmentation

### ISSUE THREAD

**Issue by rameshamurtaza** [cleanlab/cleanlab#894]
Does your active learning with single/ multi annotators support semantic segmentation?
Right now I think it only supports classification techniques

**Comment by jwmueller** [comment]
Yes the current Activelab code is specific to classification tasks.  If you'd like to help us extend it to segmentation, we would greatly appreciate that!

**Comment by MaxJoas** [comment]
@rameshamurtaza Are you still working on this? I have experience with active learning and semantic segmentation and would like to contribute.

**Comment by jwmueller** [comment]
@MaxJoas I don't think anybody is working on this, would love to see your contribution!

**Comment by MaxJoas** [comment]
@jwmueller Sounds great, I'll start working on this! Just a quick question beforehand: What input format would be most helpful for cleanlab users? I would start with allowing probmaps (n_classes, width, height) and maybe also support the output formats of the most common segmentation tools YOLO, RCNN, etc. What do you think?

**Comment by jwmueller** [comment]
> @jwmueller Sounds great, I'll start working on this! Just a quick question beforehand: What input format would be most helpful for cleanlab users? I would start with allowing probmaps (n_classes, width, height) and maybe also support the output formats of the most common segmentation tools YOLO, RCNN, etc. What do you think?

Wonderful!  Yes your plan sounds good to me, would just focus on one format for the initial PR. 
Ideally the format that best matches cleanlab's existing support for segmentation:

https://docs.cleanlab.ai/stable/tutorials/segmentation.html
